src/pke/examples/simple-ckks-bootstrapping.cpp

```diff
@@ -0,0 +1,202 @@
+
+/*
+
+Example for CKKS bootstrapping
+
+*/
+
+#define PROFILE
+
+#include "openfhe.h"
+#include "scheme/ckksrns/cryptocontext-ckksrns.h"
+#include "gen-cryptocontext.h"
+
+using namespace std;
+using namespace lbcrypto;
+
+int main(int argc, char* argv[]) {
+    // Step 1: Set CryptoContext
+    CCParams<CryptoContextCKKSRNS> parameters;
+
+    // A. Specify main parameters
+    /*  A1) Secret key distribution
+    * The secret key distribution for CKKS should either be SPARSE_TERNARY or UNIFORM_TERNARY.
+    * The SPARSE_TERNARY distribution was used in the original CKKS paper,
+    * but in this example, we use UNIFORM_TERNARY because this is included in the homomorphic
+    * encryption standard.
+    */
+    SecretKeyDist secretKeyDist = UNIFORM_TERNARY;
+    parameters.SetSecretKeyDist(secretKeyDist);
+
+    /*  A2) Desired security level based on FHE standards.
+    * In this example, we use the "NotSet" option, so the example can run more quickly with
+    * a smaller ring dimension. Note that this should be used only in
+    * non-production environments, or by experts who understand the security
+    * implications of their choices. In production-like environments, we recommend using
+    * HEStd_128_classic, HEStd_192_classic, or HEStd_256_classic for 128-bit, 192-bit,
+    * or 256-bit security, respectively. If you choose one of these as your security level,
+    * you do not need to set the ring dimension.
+    */
+    parameters.SetSecurityLevel(HEStd_NotSet);
+    parameters.SetRingDim(1 << 12);
+
+    /*  A3) Key switching parameters.
+    * By default, we use HYBRID key switching with a digit size of 3.
+    * Choosing a larger digit size can reduce complexity, but the size of keys will increase.
+    */
+    parameters.SetNumLargeDigits(3);
+    parameters.SetKeySwitchTechnique(HYBRID);
+
+    /*  A4) Scaling parameters.
+    * By default, we set the modulus sizes and rescaling technique to the following values
+    * to obtain a good precision and performance tradeoff. We recommend keeping the parameters
+    * below unless you are an FHE expert.
+    */
+#if NATIVEINT == 128
+    ScalingTechnique rescaleTech = FIXEDMANUAL;
+    usint dcrtBits               = 78;
+    usint firstMod               = 89;
+#else
+    ScalingTechnique rescaleTech = FLEXIBLEAUTO;
+    usint dcrtBits               = 59;
+    usint firstMod               = 60;
+#endif
+
+    parameters.SetScalingModSize(dcrtBits);
+    parameters.SetScalingTechnique(rescaleTech);
+    parameters.SetFirstModSize(firstMod);
+
+    /*  A4) Bootstrapping parameters.
+    * We set a budget for the number of levels we can consume in bootstrapping for encoding and decoding, respectively.
+    * Using larger numbers of levels reduces the complexity and number of rotation keys,
+    * but increases the depth required for bootstrapping.
+	* We must choose values smaller than ceil(log2(slots)).
+    */
+    std::vector<uint32_t> levelBudget = {2, 2};
```

Comment:
Typically {4,4} is reasonable for large ring dimensions

Reply:
Changed MATHBACKEND to 4 and changed to {4, 4}